### PR TITLE
CFM-922: Show status message

### PR DESCRIPTION
**Pull request by DavidHernandez**
Shows the welcome message only when the user joins the group.

Replaces the old join / request membership CTAs with the unified workflow in order to have the welcome message.

Issue: https://issuetracker.amplexor.com/jira/browse/CFM-922
Previous PR: https://github.com/capacity4dev/capacity4more/pull/1064

### CODE REVIEW

**Comment by soniCaH** [comment]
@DavidHernandez please re-roll

**Comment by ordavidil** [comment]
@DavidHernandez 
Please add screenshots ...

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module [hunk 1]
```diff
@@ -671,22 +662,23 @@ function c4m_user_og_group_actions_cta($group) {
  * of the group yet.
  */
 function c4m_user_og_c4m_join_cta($group) {
-  if (og_is_member('node', $group->nid)) {
-    drupal_set_message(
-      t('Welcome to group %group', array('%group' => $group->title))
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module]: @DavidHernandez 
Aren't we losing this message ?

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module [hunk 2]
```diff
@@ -671,22 +662,23 @@ function c4m_user_og_group_actions_cta($group) {
...
+  if (og_is_member('node', $group->nid, 'user', NULL, array(OG_STATE_PENDING))) {
+    $message = t("This is a moderated group. Your request is waiting for approval. Continue to explore group's content.");
+    drupal_set_message($message);
+
+    return array(
+      '#type' => 'markup',
+      '#markup' => '<div class="c4m-group-visitor-cta"><i class="fa fa-hourglass-end" aria-hidden="true"></i> ' . t(
+          'Membership waiting approval'
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module]: Please use one line for the whole `#markup` value.

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module (hunk 1 above)
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module]: ok, saw it down ...

**Comment by DavidHernandez** [comment]
### Desktop
status-message-desktop
### Tablet
status-message-tablet
### Mobile
status-message-mobile

**Pull request merged**